addons/payment/models/payment_acquirer.py

```diff
@@ -505,7 +581,8 @@ def _get_default_partner_country_id(self):
     reference = fields.Char(
         'Reference', default=lambda self: self.env['ir.sequence'].next_by_code('payment.transaction'),
         required=True, help='Internal reference of the TX')
-    acquirer_reference = fields.Char('Acquirer Reference', help='Reference of the TX as stored in the acquirer database')
+    acquirer_reference = fields.Char('Acquirer Reference',
+                                     help='Reference of the TX as stored in the acquirer database')
```

Comment:
Useless diff, to be repeated to the end of the file each time we avoided >80 characters lines.